dev-packages/application-package/src/application-props.ts

```diff
@@ -113,12 +113,24 @@ export interface FrontendApplicationConfig extends ApplicationConfig {
      */
     readonly applicationName: string;
 
+    /**
+     * Electron specific configuration.
+     */
+    readonly electron?: Readonly<ElectronFrontendApplicationConfig>;
+}
+
+export interface ElectronFrontendApplicationConfig extends ApplicationConfig {
+
     /**
      * If set to `true`, reloading the current browser window won't be possible with the `Ctrl/Cmd + R` keybinding.
      * It is `false` by default. Has no effect if not in an electron environment.
      */
     readonly disallowReloadKeybinding?: boolean;
 
+    /**
+     * Override or add properties to the electron `windowOptions`.
+     */
+    readonly windowOptions?: Object;
```

Comment:
are there better types than `Object`?

Reply:
Yes there is. `Electron.BrowserWindowConstructorOptions` should do the job.

Reply:
Could you update? There is new type import syntax in typescript to make sure that Electron code is erased from js.